autogpt_platform/backend/backend/util/test_json.py

```
@@ -754,3 +756,81 @@ class OuterModel(BaseModel):
         nested_metadata = metadata["nested_key"]
         assert nested_metadata == "NestedValueDelete"
+
+
+class TestValidateWithJsonschema:
+    """Test cases for validate_with_jsonschema."""
+
+    SCHEMA: dict[str, Any] = {
+        "type": "object",
+        "properties": {"name": {"type": "string"}, "age": {"type": "integer"}},
+        "required": ["name"],
+    }
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)